<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>上层管理软件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget_RTU">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>145</width>
      <height>206</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::CurrentChanged|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="itemAlignment">
     <set>Qt::AlignJustify|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <item>
     <property name="text">
      <string>RTU参数录入</string>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>外观检查、耐压试验结果</string>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>遥测误差检验</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>改变量</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>遥信变位试验</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>遥控试验</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>现场校验</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>遥控量监测</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>附加Excel报表管理</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>22</y>
      <width>131</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>67</y>
      <width>131</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>111</y>
      <width>131</width>
      <height>3</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>177</y>
      <width>131</width>
      <height>3</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_SYS">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>131</width>
      <height>402</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>true</bool>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="itemAlignment">
     <set>Qt::AlignJustify|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <item>
     <property name="text">
      <string>系统参数设置</string>
     </property>
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
    </item>
    <item>
     <property name="text">
      <string>通讯协议参数</string>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="flags">
      <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>自定义协议设置</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>设置标准装置参数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>误差检定方案管理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>改变量检定方案管理</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>仪器的通信设置</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>时钟同步</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>交流标准功率源(P)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>直流标准源</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>交流测量</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>上下载仪器软件</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>网络升级软件</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>设置工具</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>设置记录保存路径</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>设置用户</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>交流源校准</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>退出</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>89</y>
      <width>124</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>132</y>
      <width>124</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>243</y>
      <width>124</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_14">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>287</y>
      <width>124</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_15">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>353</y>
      <width>124</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
